app/modules/account/components/information.jsx

```diff
@@ -143,29 +135,49 @@ export default class AccountInformation extends React.Component {
   renderGroups() {
     return (
       <div>
-        <h1>Groups</h1>
-        {this.props.data.groups &&
-          this.props.data.groups.map(group => (
-            <dl key={group.name} className="dl-horizontal">
+        <h1>Packages</h1>
+        {this.props.data.packages &&
+          this.props.data.packages.map(item => (
+            <dl key={item.name} className="dl-horizontal">
               <dt>Name</dt>
-              <dd>{group.name}</dd>
+              <dd>{item.name}</dd>
               <dt>Expires</dt>
               <dd>
                 <Tooltip
-                  title={group.expire ? new Date(group.expire * 1000).toString() : ''}
+                  title={
+                    item.subscription &&
+                    (item.subscription.end_at || item.subscription.next_bill_at)
+                      ? new Date(
+                          item.subscription.end_at || item.subscription.next_bill_at
+                        ).toString()
+                      : ''
+                  }
                 >
-                  {group.expire ? humanize.date('F j, Y', group.expire) : 'never'}
+                  {item.subscription &&
+                  (item.subscription.end_at || item.subscription.next_bill_at)
+                    ? humanize.date(
+                        'F j, Y',
+                        new Date(
+                          item.subscription.end_at || item.subscription.next_bill_at
+                        )
+                      )
+                    : 'never'}
                 </Tooltip>
               </dd>
               <dt>Permissions</dt>
               <dd>
                 <ul>
-                  {group.permissions.map(permissionName => (
-                    <li key={permissionName}>
-                      <Icon type="check" />
-                      {' ' + permissionName}
-                    </li>
-                  ))}
+                  {Object.keys(item).map(key => {
+                    if (!key.startsWith('service.') || !item[key].title) {
+                      return null;
+                    }
```

Comment:
Just if not starts with `service.`. Please check if `type of item == object`  (I'm not sure that this is the correct syntax. Please check google)